src/Core/Services/Implementations/EventService.cs

```diff
@@ -178,24 +178,31 @@ public async Task LogPolicyEventAsync(Policy policy, EventType type, DateTime? d
         }
 
         public async Task LogOrganizationUserEventAsync(OrganizationUser organizationUser, EventType type,
-            DateTime? date = null)
+            DateTime? date = null) =>
+            await LogOrganizationUserEventsAsync(new[] {(organizationUser, type, date)});
```

Comment:
Please add spaces between the `{ }` and inner array item i.e. `{ (organizationUser, type, date) }`